Source/Core/IO/Logging/MachineLogger.cs

```diff
@@ -304,7 +304,7 @@ public virtual string FormatOnWaitString(MachineId machineId, string currStateNa
             {
                 eventNames = "[any]";
             }
-            return $"<ReceiveLog> Machine '{machineId}' in state '{currStateName}' dequeued event '{eventNames}'.";
+            return $"<ReceiveLog> Machine '{machineId}' in state '{currStateName}' is waiting to dequeue one of the events: '{eventNames}'.";
```

Comment:
@akashlal I slightly changed this to make it clear that we can be waiting for one of these events (in case the user is waiting for the first of more than one events).

Reply:
Otherwise, approved. Feel free to merge (or modify further).